lib/doc/worksheet.js

```diff
@@ -330,6 +330,10 @@ Worksheet.prototype = {
         rSrc = this._rows[i - 1];
         if (rSrc) {
           this.getRow(i + nExpand).values = rSrc.values;
+          // eslint-disable-next-line no-loop-func
+          rSrc.eachCell({ includeEmpty: true }, function(cell, colNumber) {
+            this.getRow(i + nExpand).getCell(colNumber).style = cell.style;
```

Comment:
hmm.. have we 100% that `cell.style` object hasn't multiple references from some different cells? 
may should we use deep copy here?

@cxam @alubbe what are You thinking about it?

Reply:
@Siemienik do you think that would be necessary because the row that we get the style for each cell gets set to undefined anyway?

Reply:
I worry about case like this one #669